packages/swap/src/quoting/Quoter.ts

```diff
@@ -347,4 +353,21 @@ export default class Quoter {
 
     return orders;
   }
+
+  private updateReplenishmentFactors() {
+    this.replenishmentFactors = createInternalAssetMap((asset) => {
+      const factor = this.accountIdToSocket
+        .values()
+        .reduce(
+          (acc, socket) => acc.plus(socket.data.replenishmentFactors[asset] ?? 0),
```

Comment:
i think the `replenishmentFactor` should only affect the liquidity of the account connected to the socket, no?